app/models/qdm/diagnosis.rb

```diff
@@ -0,0 +1,11 @@
+class QDM::Diagnosis < QDM::Datatype
+  include Mongoid::Document
+  field :author_datetime, type: DateTime
+  field :prevalence_period, type: QDM::Interval
+  field :anatomical_location_site, type: QDM::Code
+  field :severity, type: QDM::Code
+  field :hqmf_oid, type: String, default: '2.16.840.1.113883.10.20.28.3.110'
+  field :qrda_oid, type: String, default: '2.16.840.1.113883.10.20.24.3.135'
```

Comment:
Why do some of these models have a qrda_oid, and others do not?

Reply:
The OID list is not complete for QRDA, the field is there more to show that we will now have that info on the models.